ui/components/app/snaps/snap-avatar/snap-avatar.js

```diff
@@ -64,7 +64,7 @@ const SnapAvatar = ({
     >
       {iconUrl ? (
         <AvatarFavicon
-          backgroundColor={BackgroundColor.backgroundAlternative}
+          style={{ "background-color": 'var(--color-background-alternative-hover)' }}
```

Comment:
Why is this necessary?

Reply:
The color we wanted to use is not available in `BackgroundColor` or on the components we use 😞

Reply:
Can we add it to `BackgroundColor`?

Reply:
Yeah, but it doesn't work since there is no generated CSS class for `Box` to use this color.